admin-dev/themes/new-theme/js/pages/module/controller.js

```diff
@@ -275,6 +276,13 @@ class AdminModuleController {
     });
   }
 
+  onModuleUpgraded(event) {
+    const self = this;
+    const moduleElement = $(event.detail[0]);
+
+    console.log(`Module ${moduleElement.data('tech-name')} upgraded`)
```

Comment:
```suggestion
```

Reply:
I let it on purpose to ... do something 

I can clean it ;)

Reply:
Cleaning it means clean the event listener in this controller. Useless 😄